anasey01/BudgetMyFunds
# #171018294 Setup Code Linting and Fix those errors

#### What does this PR do?
- it introduces a lint tool that adheres to PSR4
#### Description of Task to be completed?
- add code sniffer to the project that sticks to psr4
- add a pre-commit file
#### How should this be manually tested?
- refer to the readme to setup your machine
- run `composer install` to get the new packages installed to this project
- run `composer lint` or `composer lint:fix` to discover code that needs "linting" or lint and fix run the commands as it appears earlier.
- incase you forget to run this commands, when you commit your code, a pre-commit hook kicks in and discovers the parts of the code that needs linting
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[Story ID](https://www.pivotaltracker.com/story/show/171018294)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
